scm-plugin/src/main/resources/archetype-resources/src/main/java/HelloWorldSCMChangeLogParser.java

```diff
@@ -0,0 +1,9 @@
+package $package;
+
+import hudson.scm.ChangeLogParser;
+
+public class HelloWorldSCMChangeLogParser extends ChangeLogParser {
```

Comment:
Unused here.

Reply:
That class is required for:

```
    @Override
    public ChangeLogParser createChangeLogParser() {
        return new HelloWorldSCMChangeLogParser();
    }
```

`createChangeLogParser` method is annotated with `@Nonnull`, so findbugs will fail if it is not defined.